packages/unlock-js/src/web3Service.ts

```diff
@@ -899,26 +899,6 @@ export default class Web3Service extends UnlockService {
     return await unlockContract.publicLockLatestVersion()
   }
 
-  /**
-   * Returns referrer fees
-   */
-  async referrerFees({
```

Comment:
Hum, why did you remove this?

Reply:
Good question.

Before we used to get referrer fees from the smart contract:

![image](https://github.com/unlock-protocol/unlock/assets/136458308/1fe19a81-c06d-4fb4-a8f8-d954f5c540e1)

With this update we no longer check referrer fees via smart contract. Instead we use subgraph 😁

Reply:
Let's still keep it in the lib please as someone may want to use it without using the subgraph.